src/modules/assessments/components/household/HouseholdAssessments.tsx

```diff
@@ -70,7 +71,7 @@ const calculateAssessmentStatus = (
 const HouseholdAssessments: React.FC<Props> = ({
   role,
   enrollment,
-  assessmentId,
+  formDefinition,
```

Comment:
note to self... we can also get rid of the "always mounted tab panel" approach

Reply:
I'm not sure if this was the right approach, maybe it would be better if each tab fetched their own thing and it was cached since they're the same. Ill do a separate PR for the tab panel mounting change